wfd/parser/videoformats.cpp

```diff
@@ -127,10 +127,9 @@ std::vector<EnumType> MaskToEnumList(ArgType from, EnumType biggest_value) {
 
   while (copy != 0) {
     if ((copy & 1) != 0) {
-      if (enum_value > static_cast<unsigned>(biggest_value)) {
-        assert(false);
+      if (enum_value > static_cast<unsigned>(biggest_value))
         break;
-      }
```

Comment:
I would have added debug logging but we don't seem to have any support for that in wfd. I thought GLib logging (that @alexshalamov used in mirac_network) was a fine idea ... but it doesn't really make sense in wfd that doesn't depend on GLib: we can't add that dependency for just logging. I guess I should add a macro that depends on CMAKE_BUILD_TYPE...